guide/java/archetype.md

```diff
@@ -26,11 +26,13 @@ Alternatively, all options can be supplied at the command line. For example,
 if creating a project named "autobrick" for "com.acme":
 
 {% highlight bash %}
+# BROOKLYN_VERSION={{ site.brooklyn-version }}
 $ mvn archetype:generate \
 	-DarchetypeGroupId=org.apache.brooklyn \
 	-DarchetypeArtifactId=brooklyn-archetype-quickstart \
-	-DarchetypeVersion={{ site.brooklyn-version }} \
-	-DgroupId=com.acme -DartifactId=autobrick \
+	-DarchetypeVersion=${BROOKLYN_VERSION} \
```

Comment:
Making this an env variable makes the example non-copy-pasteable. The user needs to edit it before executing.
Prefer the old version, or if you really want to make the version stand out change to something like:

```
$ BROOKLYN_VERSION={{ site.brooklyn-version }} \
   mvn archetype:generate \
	-DarchetypeGroupId=org.apache.brooklyn \
	-DarchetypeArtifactId=brooklyn-archetype-quickstart \
	-DarchetypeVersion=${BROOKLYN_VERSION} \
	-DgroupId=com.acme \
	-DartifactId=autobrick \
	-Dversion=0.1.0-SNAPSHOT \
	-DpackageName=com.acme.autobrick \
	-DinteractiveMode=false
```